src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -311,6 +311,85 @@ pre-warmed `FlutterEngine` based on the ID given to `withCachedEngine()`.
 By providing a pre-warmed `FlutterEngine`, as previously shown, your app renders the
 first Flutter frame as quickly as possible.
 
+#### Initial route with a cached engine
+
+The concept of an initial route is available when configuring a 
+`FlutterActivity` or a `FlutterFragment` with a new `FlutterEngine`. However,
+`FlutterActivity` and `FlutterFragment` do not offer the concept of an initial
```

Comment:
Nit:
```suggestion
`FlutterActivity` and `FlutterFragment` don't offer the concept of an initial
```
One reason that such contractions are useful: it's sometimes easy for a reader to miss the word "not," whereas it's harder to misread "don't" as "do."

See https://developers.google.com/style/contractions